frontend/src/App.css

```diff
@@ -432,8 +354,8 @@ select {
   padding: 0.5rem 1rem;
   border-radius: 2rem;
   font-weight: 800;
-  margin-right: 6rem;
-  margin-left: 6rem;
+  padding-right: 3rem;
+  padding-left: 3rem;
```

Comment:
you're setting padding to `0.5rem 1rem` then overwrite it for left and right. try :

    padding: 0.5rem 3rem;